src/js_native_api.h

```diff
@@ -118,6 +118,22 @@ NAPI_EXTERN napi_status NAPI_CDECL node_api_create_property_key_utf16(
     napi_env env, const char16_t* str, size_t length, napi_value* result);
 #endif  // NAPI_EXPERIMENTAL
 
+#ifdef NAPI_EXPERIMENTAL
+#define NODE_API_EXPERIMENTAL_HAS_PROPERTY_KEYS
```

Comment:
Defining the same macro twice will cause compiler warnings or errors. Please group the new `node_api_create_property_key_utf8` API with the previous `node_api_create_property_key_utf16`.